.gitignore

```diff
@@ -128,6 +128,10 @@ ENV/
 env.bak/
 venv.bak/
 
+# Handling Flask Environment Variables
+# Faciliates debugging code errors
+FLASK_DEBUG = True
```

Comment:
This should be `.flaskenv` :) The _file_ should be gitignored. This config should still be in your local `.flaskevn`